rds-monitoring/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Default values for rds-monitoring.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+## The following settings are for the prometheus-cloudwatch-exporter chart
+prometheus-cloudwatch-exporter:
+  replicaCount: 1
+
+  serviceMonitor:
+    enabled: true
+    interval: 60s
+    labels:
+      prometheus: cloudwatch-exporter
+    timeout: 30s
+
+  resources:
+    limits:
+    memory: 320Mi
+  requests:
+    cpu: 10m
+    memory: 320Mi
+
+  aws:
+    region: eu-west-1
+    role:
+
+  serviceAccount:
+    create: true
+
+  rbac:
+    create: true
+
+  config: |-
+    region: eu-west-1
+    period_seconds: 60
+    set_timestamp: false
+    metrics:
+      - aws_namespace: AWS/RDS
+        aws_metric_name: CPUCreditBalance
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: CPUUtilization
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: FreeableMemory
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: FreeStorageSpace
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: DiskQueueDepth
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: ReadIOPS
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
+      - aws_namespace: AWS/RDS
+        aws_metric_name: WriteIOPS
+        aws_dimensions: [DBInstanceIdentifier]
+        aws_statistics: [Minimum, Maximum, Average, Sum]
```

Comment:
i'm missing replica lag

i'm also not sure if all of these are compatible with Aurora

Reply:
I've added the ReplicaLag metric.
These metrics are fetched "as is" from cloudwatch, so if an RDS instance doesn't generate a specific metric, it simply won't be fetched by the exporter.